spec/requirements.html

```diff
@@ -2,6 +2,6 @@
   <h1>Requirements for Standard Built-in ECMAScript Objects</h1>
 
   <p>
-    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor), clause <emu-xref href="#sec-ecmascript-standard-built-in-objects"></emu-xref>.
+    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2024 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor), clause <emu-xref href="#sec-ecmascript-standard-built-in-objects"></emu-xref>.
```

Comment:
```suggestion
    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2024 Language Specification (ECMA-262 15<sup>th</sup> Edition, or successor), clause <emu-xref href="#sec-ecmascript-standard-built-in-objects"></emu-xref>.
```